sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -1075,9 +1113,15 @@ export interface StorageDirectoryRenamedEventData {
     storageDiagnostics?: any;
 }
 
+// @public
+export interface StorageLifecyclePolicyActionSummaryDetail {
+    errorList?: string;
+    successCount?: number;
+    totalObjectsCount?: number;
+}
+
 // @public
 export interface StorageLifecyclePolicyCompletedEventData {
-    // Warning: (ae-forgotten-export) The symbol "StorageLifecyclePolicyActionSummaryDetail" needs to be exported by the entry point index.d.ts
```

Comment:
This line of code change is not really related to Version Upgrade. I just found a forgotten export and added it. Please let me know if this is not exported intentionally. I can revert the changes. Thanks